darts/models/forecasting/forecasting_model.py

```diff
@@ -307,6 +308,8 @@ def historical_forecasts(
         num_samples
             Number of times a prediction is sampled from a probabilistic model. Should be left set to 1
             for deterministic models.
+        train_length
+            Number of time steps in our training set (size of backtesting window to train on).
```

Comment:
We could mention that it uses all available time steps up until prediction time if set to `None`.

Reply:
+1. Also you can explain that the expanding window strategy is used by default with train_length=None, and otherwise the moving window strategy is used.